libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -35,10 +36,10 @@ class CreatorNodeSelection extends ServiceSelection {
         })
       },
       // Use the content node's configured whitelist if not provided
-      whitelist: whitelist || creatorNode.passList,
-      blacklist: blacklist || creatorNode.blockList
+      whitelist: whitelist || (creatorNode && creatorNode.passList),
+      blacklist: blacklist || (creatorNode && creatorNode.blockList)
     })
-    this.creatorNode = creatorNode
+    this.creatorNode = creatorNode || new CreatorNode()
```

Comment:
what are the code changes in this file for?

Reply:
discussed offline: keep lines 39, 40, 248